src/Files.App/Views/Settings/DevToolsPage.xaml

```diff
@@ -76,5 +246,38 @@
 			</local:SettingsBlockControl>
 
 		</StackPanel>
+
+		<VisualStateManager.VisualStateGroups>
+			<VisualStateGroup>
+				<VisualState x:Name="DefaultState">
```

Comment:
If the main goal is to adjust the browse button's position, we can simplify the visual states by using a StackPanel and changing the orientation based on the page width.

Reply:
I'm using a grid to place elements in the read-only UI, which flows in both directions.
I think that removing the grid would require other 2/3 stackpanels.

BTW I found there are a couple of states to be removed (`PickIDEExe.(Grid.*)`)

Reply:
👍